spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -184,6 +188,12 @@ def get_parser():
         metavar=Metavar.folder,
         action=ActionCreateFolder,
         help="The path where the quality control generated content will be saved."
+             "QC report is only availbale for PMJ-based CSA."
+    )
+    optional.add_argument(
+        '-qc-image',
+        metavar=Metavar.str,
+        help='Input image to display in QC report. To be used with flag -qc '
```

Comment:
```suggestion
        help="Input image to display in QC report. Typically, it would be the image that was used for spinal cord segmentation. This flag is mandatory if using flag '-qc'."
```

Reply:
Ahh, this is a good clarification!

With this new wording, though, I wonder if people might misread it and accidentally supply the segmented spinal cord here too. This wording might be a little more explicit:

```suggestion
        help="Input image to display in QC report. Typically, it would be the "
             "source anatomical image used to generate the spinal cord "
             segmentation. This flag is mandatory if using flag '-qc'."
```